fb_mcbot/views.py

```diff
@@ -35,20 +37,35 @@ def get_user_info(user_id):
     user = requests.get(("https://graph.facebook.com/v2.6/%s?access_token=%s" % (user_id, PAGE_ACCESS_TOKEN)))
     return user
 
-def userExists(userid):
-    try:
-        user = FBUser.objects.get(user_id = userid)
-    except FBUser.DoesNotExist:
-        pprint("User id not found in db, the user does not exist.")
-        return False;
-    return True;
-
-def create_new_user(user,user_id):
-    pprint("Creating new user")
-    firstname = user['first_name']
-    lastname = user['last_name']
-    new_user = FBUser(first_name = firstname, last_name = lastname, user_id =  user_id)
-    new_user.save()
+    #business logic
+def logic(request, message):
```

Comment:
Change the name to something more descriptive like process_message, or whatever you feel is more appropriate. Also, it doesn't seem to use its first argument "request". Finally, it should be moved to another file (in bot_services most likely) since there shouldn't be business logic in views. Same for the other functions that aren't just helpers.